src/main/java/wolox/training/repositories/BookRepository.java

```diff
@@ -0,0 +1,22 @@
+package wolox.training.repositories;
+
+import java.util.Optional;
+import org.springframework.data.repository.Repository;
+import wolox.training.models.Book;
+
+/**
+ * BookRepository.java Interface that serves as a Repository for the Book model.
```

Comment:
As a convention, the class name is not included in the Javadoc (To see this Javadoc you should get to the class, so you already know from where it belongs)